src/Foundation/NSUuid.cs

```diff
@@ -40,11 +40,12 @@ unsafe public NSUuid (byte [] bytes) : base (NSObjectFlag.Empty)
 		public byte [] GetBytes ()
 		{
 			byte [] ret = new byte [16];
-			
-			IntPtr buf = Marshal.AllocHGlobal (16);
-			GetUuidBytes (buf);
-			Marshal.Copy (buf, ret, 0, 16);
-			Marshal.FreeHGlobal (buf);
+
+			unsafe {
```

Comment:
This one doesn't seem to match the others in description, though it looks reasonable to me.

Reply:
I think this was supposed to be in a different PR. I can split it out if you want.